sw/device/tests/clkmgr_external_clk_src_for_sw_test.c

```diff
@@ -0,0 +1,138 @@
+// Copyright lowRISC contributors.
```

Comment:
Shouldn't this go in `sw/device/tests/sim_dv/` since it only works in the DV sim environment currently?

Reply:
I don't follow... this test is software only. The only thing it relies on is the "clk" input to the chip. It could also be run in the FPGA, provided that implements the extclk interface.

What are the guidelines for placing tests in sim_dv?

I will commit this, but it is no problem to move it elsewhere in a separate PR as needed.

Reply:
Ah right, sorry I mistook the `chip_base_vseq.sv` for a test-specific sequence. This is good where it is. It should be portable to other HW platforms provided things are wired up correctly. My apologies!

But to answer your question, generally, if this test has a custom SV host-side component (i.e. **_beyond_** just `chip_base_vseq.sv`) it should go in sim_dv/.